### DISCUSSION TITLE
App runs in VS debug fine but crashes immediately when opened on Android phone

### DISCUSSION THREAD

**Discussion by nickj886** [dotnet/maui#6461] [Q&A]
Hi,

I am writing my first Maui app and it works fine in Visual Studio but it closes immediately without a message on opening it on my Android phone.

Is there any way of locating log any system logs for my app in my phone?

Regards
Nick

**Comment by xtuzy** [comment]
You can get log at VisualStudio>Tool>Android>Device Log. It run well at emulator? 

![sharex-20220425105028](https://user-images.githubusercontent.com/17793881/165012786-2bfd96a5-e0e3-4c1d-9475-a47cbd182ba5.png)

**Reply by nickj886** [reply]
Hi @xtuzy

Thanks for the tip on the Device Log feature in VS. It was very helpful for seeing the activity going on in the currently debugged device (emulator or physical).

After a bit more trial and error I found that the APK that I was generating via the command line with "dotnet build" was different in size than the APK generated when I built the project in VS. The VS built APK worked perfectly where my command line APK didn't.

Cheers
Nick
